### PR TITLE
Fix projection of analog polls & prevent percent base 'entitled' for analog polls

### PR DESCRIPTION
**Pull request by jsangmeister**

### CODE REVIEW

File: server/openslides/poll/serializers.py [hunk 1]
```diff
@@ -136,7 +141,17 @@ def validate(self, data):
             )
         return data
 
-    def norm_100_percent_base_to_pollmethod(
-        self, onehundred_percent_base, pollmethod, old_100_percent_base=None
+    def norm_100_percent_base(
+        self, onehundred_percent_base, pollmethod, polltype, old_100_percent_base=None
     ):
-        raise NotImplementedError()
+        if (
```
Comment (FinnStutzenstein) [inline server/openslides/poll/serializers.py:147]: Looks  like this needs to be ported to OS4 as well.

Comment (jsangmeister) [reply server/openslides/poll/serializers.py:147]: Yes, I'll open issues for it

**Review by tsiegleauq — APPROVED** [review]
Code looks fine. please redo in os4client or create an issue

**Comment by jsangmeister** [comment]
> please redo in os4client or create an issue

Already did, see referencing issues above. Or do you mean an extra one for the projection part? I can create another issues for that, if you wish

**Pull request merged**